src/displayapp/screens/StopWatch.cpp

```diff
@@ -119,26 +142,29 @@ void StopWatch::Refresh() {
     TimeSeparated_t currentTimeSeparated = convertTicksToTimeSegments(laps[lapsDone]);
     lv_label_set_text_fmt(time, "%02d:%02d", currentTimeSeparated.mins, currentTimeSeparated.secs);
     lv_label_set_text_fmt(msecTime, "%02d", currentTimeSeparated.hundredths);
+  } else if (currentState == States::Halted) {
+    if (xTaskGetTickCount() > blinkTime) {
+      blinkTime += blinkInterval;
+      if (lv_obj_get_state(time, LV_LABEL_PART_MAIN) == LV_STATE_DEFAULT) {
+        lv_obj_set_state(time, LV_STATE_DISABLED);
+        lv_obj_set_state(msecTime, LV_STATE_DISABLED);
+      } else {
+        lv_obj_set_state(time, LV_STATE_DEFAULT);
+        lv_obj_set_state(msecTime, LV_STATE_DEFAULT);
+      }
+    }
   }
 }
 
-void StopWatch::playPauseBtnEventHandler(lv_event_t event) {
-  if (event != LV_EVENT_CLICKED) {
```

Comment:
why are the events removed? Doesn't this mean a long press event could start the timer as well as a normal click?

Reply:
The if-statement has just been moved to the calling function.

Reply:
ah thanks!